src/Alba/HeaderExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Net.Http.Headers;
+
+namespace Alba
+{
+    public static class HeaderExtensions
+    {
+        public static long? GetContentLength(this IHeaderDictionary headers)
```

Comment:
Just to be ticky tacky, I've been using the naming convention like `long? ContentLength()` and `ContentLength(long length)` in the rest of the codebase. I think we do need to be consistent one way or another, and I can be convinced to go the Get/Set naming route.